data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```
@@ -60,14 +60,18 @@
         "O=NotaryService, L=London, C=GB"
       ]
     },
-    "corda.notary.service.plugin": {
-      "description": "The notary plugin type (FQN of notary service plugin class). Valid only when one of the roles is notary.",
+    "corda.notary.service.flow.protocol.name": {
+      "description": "Name of the flow protocol used by the notary service. Valid only when one of the roles is notary.",
       "type": "string",
       "maxLength": 255,
       "examples": [
         "net.corda.notary.MyNotaryService"
       ]
     },
+    "^corda.notary.service.flow.protocol.versions.[0-9]+$": {
```

```suggestion
    "^corda.notary.service.flow.protocol.version.[0-9]+$": {
```
Property refers to a single version